PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalLauncher.kt

```diff
@@ -87,6 +106,25 @@ class PayPalLauncher internal constructor(private val browserSwitchClient: Brows
         }
     }
 
+    @OptIn(ExperimentalBetaApi::class)
+    fun handleReturnToAppFromBrowser(
+        pendingRequest: PayPalPendingRequestEditFi.Started,
+        intent: Intent
+    ): PayPalVaultEditAuthResult {
+        return when (val browserSwitchResult =
+            browserSwitchClient.completeRequest(intent, pendingRequest.pendingRequestString)) {
+            is BrowserSwitchFinalResult.Success -> PayPalVaultEditAuthResult.Success(
+                PayPalVaultEditAuthResultInfo(browserSwitchResult)
```

Comment:
We shouldn't need this `PayPalVaultEditAuthResultInfo` wrapper class anymore - the `PayPalVaultEditAuthResult.Success` class can just wrap the `browserSwitchResult` see [this example](https://github.com/braintree/braintree_android/blob/main/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalLauncher.kt#L72)